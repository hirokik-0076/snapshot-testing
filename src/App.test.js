import React from 'react'
import App from './App'
import renderer from 'react-test-renderer'

it('matches to the snapshot', () => {
  const tree = renderer.create().toJSON()
  expect(tree).toMatchSnapshot()
})
