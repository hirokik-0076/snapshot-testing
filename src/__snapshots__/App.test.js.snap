// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches to the snapshot 1`] = `null`;
